Form/Type/VichImageType.php

```diff
@@ -90,4 +106,21 @@ public function getBlockPrefix(): string
     {
         return 'vich_image';
     }
+
+    private function resolvePath($storageResolveMethod, $object, FormInterface $form): ?string
```

Comment:
can we type-hint? E.g. `int $storageResolveMethod, object $object`)

Reply:
I thought about it, but noticed that in `VichFileType` there was a similar method with no type hints. So I restrained, not knowing if there was a good reason for that. Could you confirm we want these type hints ? And should I add them in the other method too while I'm at it?

Reply:
I'd prefer such changes in files that need to be changed for other reasons (like this one). We'll handle other files when needed (or in a PR dedicated to bring consistency)